edge-modules/SimulatedTemperatureSensor/src/Program.cs

```diff
@@ -74,19 +74,6 @@ static async Task<int> MainAsync()
             (CancellationTokenSource cts, ManualResetEventSlim completed, Option<object> handler) = ShutdownHandler.Init(TimeSpan.FromSeconds(5), null);
 
             Twin currentTwinProperties = await moduleClient.GetTwinAsync();
-            if (currentTwinProperties.Properties.Desired.Contains(SendIntervalConfigKey))
```

Comment:
why remove it? this is what e2e test runs now.

Reply:
We should discuss in the meeting.

Reply:
We should revert.